l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py

```diff
@@ -1102,6 +1102,16 @@ def test_duplicated_vat_on_partners(self):
         supplier = self.env["res.partner"].search(
             [("vat", "=", "IT05979361218")], limit=1
         )
+        if not supplier:
+            # do not depend on former tests to be executed prior to this one
+            res = self.run_wizard("duplicated_vat1", "IT05979361218_001.xml")
+            invoice_id = res.get("domain")[0][2][0]
+            invoice = self.invoice_model.browse(invoice_id)
+            invoice.ref = invoice.payment_reference = "14dupvat"
+            supplier = self.env["res.partner"].search(
```

Comment:
Quindi per assurdo, questo test non è coperto dai test!
Non si potrebbe invece usare un'altra partita IVA che non è ancora stata usata per questo partner? Ho trovato https://strumentidev.it/generatore/partita-iva per generarne di valide.

Ho usato questa soluzione in https://github.com/OCA/l10n-italy/pull/3810/commits/b551127b51c17b02c33fa9c97f3a7b343d79f42b (di https://github.com/OCA/l10n-italy/pull/3810)